searchcore/src/apps/vespa-proton-cmd/vespa-proton-cmd.cpp

```diff
@@ -158,6 +159,9 @@ class App : public FastOS_Application
         } catch (config::InvalidConfigException& e) {
             fprintf(stderr, "ERROR: failed to get service location broker configuration\n");
             std::_Exit(1);
+        } catch (vespalib::IllegalStateException& e) {
```

Comment:
consider using a more specific exception, this could be caused by 'anything'

Reply:
I concluded that this would be fine for now as it is an improvement of something that should not really happen.
This is just to avoid the core dump and provide some hint at why it exited.
I added printing of exact exception info too.